### DISCUSSION TITLE
Calculating biasing forces from Well-tempered Metadynamics simulations

### DISCUSSION THREAD

**Discussion by Praveen-Prabhakar** [Colvars/colvars#680] [Q&A] [answered]
Hi all, 

I have run a well-tempered metadynamics simulations on NAMD with colvars for biasing the CV. I am writing to ask if there is any COLVARS module/method which can calculate the biasing force applied from the .dcd, or .hills.traj file? I didn't save forces during the simulation run though!

Thanks in advance.

**Comment by fabsugar** [comment, marked as answer]
Hi,
there is a way to do that using the FCAM tool (
https://github.com/FCAM-NIH/FCAM). You need the .colvars.traj and the
.hills.traj files. If you don't have the .colvars.traj file you can
calculate it from the .dcd using the colvars dashboard. Please check the
FCAM manual for doing that (there are also examples in the tutorial folders
which are also explained in the manual).
Hope it helps,
Fabrizio

**Comment by Praveen-Prabhakar** [comment]
Hi Fabrizio! 

Thank you for sharing this! Yes I will got through the tutorial.
